sdk/generic-sdk-core/generic-core/src/test/java/com/generic/core/implementation/util/InternalContextTests.java

```diff
@@ -1,36 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-package com.generic.core.implementation.util;
-
-import com.generic.core.util.ClientLogger;
-import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.MethodSource;
-
-import java.util.Collections;
-import java.util.stream.Stream;
-
-import static org.junit.jupiter.api.Assertions.assertSame;
-
-/**
- * Tests for {@link InternalContext}.
- */
-public class InternalContextTests {
```

Comment:
Was this copied when generic-core was renamed to client-core?

Reply:
Yes, you can find it on `client-core` [here](https://github.com/Azure/azure-sdk-for-java/tree/feature/generic-sdk-core-2/sdk/client-core/core/src/main/java/io/clientcore/core/implementation/util).